src/components/BreedsList.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <section>
+    <div v-for="breed in breedsList">
+      <h3>{{ breed.name }}</h3>
+      <span>{{ breed.breed_group }}</span>
+      <span>{{ breed.life_span }}</span>
+      <img
+        class="breedsList_image"
+        :src="breed.image.url"
+        :alt="`Picture of the ${breed.name} dog`"
+      />
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  props: ["name"],
+  data() {
+    return {
+      breedsList: [],
+    };
+  },
+  methods: {
+    fetchAllData() {
+      fetch(`https://api.thedogapi.com/v1/breeds/`)
+        .then((response) => response.json())
+        .then((results) => {
+          this.breedsList = results;
+        })
+        .catch((error) => console.log(error));
+    },
+  },
+  mounted() {
```

Comment:
teoretično se lahko api klici kličejo tud v `mounted` ampak je boljš da se data pofetcha v `created` hooku in s tem se lahko vse nastavi preden se komponenta mounta in se že mounta z dato, namesto da se šele pol pokliče api in pridobijo podatki.. ok?

Reply:
mogoče bi blo dobr narest nek `data.js` file in vse api klice prestavit v ta file, kjer pol samo returnaš promise in podatke v komponento.. zato da se loči UI componenta in pure data del

Reply:
> teoretično se lahko api klici kličejo tud v `mounted` ampak je boljš da se data pofetcha v `created` hooku in s tem se lahko vse nastavi preden se komponenta mounta in se že mounta z dato, namesto da se šele pol pokliče api in pridobijo podatki.. ok?

Ok, bom spremenil. Pol tudi v bodoče delam, da se api klice pofetcha kar v `created` hooku?

Reply:
ja